js/sts-hotspot.js

```diff
@@ -0,0 +1,45 @@
+AFRAME.registerComponent('sts-hotspot', {
+  schema: {
+    show: { type: 'selector' },
+    sound: { type: 'selector' }
+  },
+
+  init: function () {
+    var self = this;
+
+    self.el.addEventListener('mouseenter', function (e) {
+      self.playHotspotSound();
```

Comment:
this is significantly better than loading all the `<audio>` elements on load in `<a-assets>`.

there is a noticeable delay when eye-gazing on a hotspot and the audio doesn't play immediately. would it be possible to lazy-load the sounds on the `sceneEl`'s `'enter-vr'` event, using [`window.requestIdleCallback`](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback)?

Reply:
That's what the `<a-assets>` is suppose to manage, but doesn't quite work.  If you look at the network tab in inspector, the request is made, but the actual asset isn't downloaded.  Hence the explicit `.load()` call.  It's not a great solution, but it works. 

There could be some mobile specific mobile network handling at play here since this issue doesn't manifest itself anywhere except FXR.  Works fine on desktop.

cc @bluemarvin

Reply:
there is a bug with `<a-assets>` it sounds like. if it's not filed, it probably should be.

moving forward, this should be fine for now.  filed issue #84 to use `window.requestIdleCallback` to [**load**](https://github.com/EFForg/spot_the_surveillance/pull/70/files/12de8fdd36b293ba689b825545c6eb3c3f615b03#diff-f34d5dddff844edea04f51bf4797b4a2R27) (and not yet play) the sounds, instead of loading it on `mouseenter`.

Reply:
Good call.